fv3core/fv3core/grid/geometry.py

```diff
@@ -601,119 +601,6 @@ def calculate_grid_a(z11, z12, z21, z22, sin_sg5):
     return a11, a12, a21, a22
 
 
-def edge_factors(
```

Comment:
this calculation is moved to a2b_ord4

Reply:
Basically did this due to a comment in a2b that the edge factors should be computed locally in that module because it is the only place it is used. After our discussion, have reverted this change, and a future PR may put them in their own class structure to make it easier to read.